GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -11,6 +12,8 @@ namespace GVFS.Platform.Mac
     public class ProjFSKext : IKernelDriver
     {
         private const string DriverName = "io.gvfs.PrjFSKext";
+        private const int LoadKext_ExitCode_Success = 0;
+        private const int LoadKext_ExitCode_NotApproved = 27;

```

Comment:
Where did you find these values?

Reply:
https://developer.apple.com/library/archive/technotes/tn2459/_index.html